### ISSUE TITLE
When KODI extension/app?

### ISSUE THREAD

**Issue by Sajjon** [kaegi/alass#6]
This is awesome! Have you considered building a small KODI app using this?

**Comment by kaegi** [comment]
Yes, I have! But for now, I want to focus on other projects. If somebody other wants to pick up the development of such an extension for their favorite video player, please do so!

For me there are two projects I'd really like to see this integrated: mpv (my favorite video player), KODI (very useful in general).

**Comment by skorokithakis** [comment]
That'd be fantastic. I wonder if mpv can be scripted to run an external utility (since I wouldn't want to run alass every time), I'll have to look into that.

**Comment by sitolam** [comment]
+1
